production/promtail-mixin/dashboards.libsonnet

```diff
@@ -4,23 +4,22 @@ local loki_mixin_utils = import 'loki-mixin/dashboards/dashboard-utils.libsonnet
 local utils = import 'mixin-utils/utils.libsonnet';
 
 {
-  grafanaDashboards+: {
+  grafanaDashboards+:: {
```

Comment:
This puts the mixin in-line with other mixins. Without it I cannot locally build with Grizzly.